controllers/drplacementcontrol.go

```diff
@@ -2296,6 +2296,13 @@ func (d *DRPCInstance) shouldUpdateStatus() bool {
 		return true
 	}
 
+	if vrg.Status.KubeObjectProtection.CaptureToRecoverFrom != nil {
+		vrgKubeObjectProtectionTime := vrg.Status.KubeObjectProtection.CaptureToRecoverFrom.EndTime
```

Comment:
(nit) Is there a reason for a local variable here?

Reply:
This is only to keep the line within 120 columns.